src/hotspot/share/runtime/flags/jvmFlag.hpp

```diff
@@ -290,6 +291,23 @@ class JVMFlag {
 
   static const char* flag_error_str(JVMFlag::Error error);
 
+  // type checking
+#define CHECK_COMPATIBLE(type) \
```

Comment:
CHECK_COMPATIBLE doesn't appear to be used outside the following assertion function.  Add an `#undef` afterward.